TinyConsole/TinyConsole.swift

```diff
@@ -14,8 +14,8 @@ open class TinyConsole {
     var consoleController: TinyConsoleController
     
     public static var textAppearance = [
-        NSFontAttributeName: UIFont(name: "Menlo", size: 12.0)!,
-        NSForegroundColorAttributeName: UIColor.white]
+        NSAttributedString.Key.font: UIFont(name: "Menlo", size: 12.0)!,
```

Comment:
You can also just write .font and .foregroundColor here.

Reply:
Thanks @ohitsdaniel! I used the migrator and it didn't catch that